src/main/java/teammates/common/datatransfer/FeedbackSessionLogEntry.java

```diff
@@ -4,19 +4,25 @@
  * Represents a log entry of a feedback session.
  */
 public class FeedbackSessionLogEntry implements Comparable<FeedbackSessionLogEntry> {
+    private final String courseId;
     private final String studentEmail;
     private final String feedbackSessionName;
     private final String feedbackSessionLogType;
     private final long timestamp;
 
-    public FeedbackSessionLogEntry(String studentEmail, String feedbackSessionName,
-            String feedbackSessionLogType, long timestamp) {
+    public FeedbackSessionLogEntry(String courseId, String studentEmail,
```

Comment:
@cedricongjh Should GCP log the student's email? I think you mentioned that email is possible to change. If email changes, may break the updating action in future

Reply:
ah yes, good point, we should also change the log's format, to log `studentId` and `feedbackSessionId` as well, this way, when creating the Entity in SQL, we would only need to query by Id.

For `GetFeedbackSessionLogsAction`, we should be using `studentId` and `feedbackSessionId` too (instead of `email` and `feedbackSessionName`)

This will require FE changes (to change the logs format, as well as API request params), so let's change the BE here first and do the FE in a separate PR